performance/collector/cron_job.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
This is architecturally wrong. 

What we actually want is to have Docker Image based on Rally Docker Image (latest release) 
In our image we should just add plugin for Rally in proper place and have entrypoint that executes cron job that runs Rally. do not create nested containers.